cmd/frontend/internal/httpapi/internal.go

```diff
@@ -202,13 +205,25 @@ func serveSearchConfiguration(db dbutil.DB) func(http.ResponseWriter, *http.Requ
 		}
 		repoNames := r.Form["repo"]
 
+		indexedIDs := make([]api.RepoID, 0, len(r.Form["repo-ids"]))
```

Comment:
the previous field was called `repo`, because it contained just a single repo, but was specified multiple times. Should we rename it for the new path, too?

```suggestion
		indexedIDs := make([]api.RepoID, 0, len(r.Form["repoID"]))
```

Reply:
will update